pkg/kv/kvserver/replica_rangefeed.go

```diff
@@ -656,10 +657,12 @@ func (r *Replica) handleClosedTimestampUpdateRaftMuLocked(
 		// Ignore the result of DoChan since, to keep this all async, it always
 		// returns nil and any errors are logged by the closure passed to the
 		// `DoChan` call.
-		_, _ = m.RangeFeedSlowClosedTimestampNudge.DoChan(key, func() (interface{}, error) {
+		taskCtx, sp := tracing.EnsureForkSpan(ctx, r.AmbientContext.Tracer, key)
```

Comment:
Would it be marginally better to make the forked span inside of the task? (So that non-leaders don't make an extra span)? Or is that not possible because of use-after-finish?